README.md

```diff
@@ -170,6 +170,18 @@ lspconfig.tsserver.setup({
             -- inlay hints
             auto_inlay_hints = true,
             inlay_hints_highlight = "Comment",
+            inlay_hints_priority = 200, -- priority of the hint extmarks
+            inlay_hints_throttle = vim.o.updatetime, -- throttle the inlay hint request, set to 0 for no throttle
+            inlay_hints_format = { -- format options for individual hint kind
+                Type = {
+                    highlight = nil,
+                    text = function(text)
```

Comment:
I like the idea of exposing this option, but I personally find the arrow syntax a little confusing (the distinction between `->` and `=>` is a little hard to parse at a glance). Is this based on something or just personal preference? If it's the latter, I think I'd prefer to leave this as `nil` by default but document the arrow as a potential option.

Reply:
It might depend on the font used. It looks easy to distinguish between them for the font I use. I originally think it's a better default but since that's not good enough we should simply document that as an example.

Reply:
This is what it looks like using PragmataPro:

<img width="579" alt="Screen Shot 2021-12-29 at 1 41 01 PM" src="https://user-images.githubusercontent.com/54108223/147693469-6e33c529-6a21-4e9e-b8ff-127d753bdef1.png">

And this is with Iosevka:

<img width="579" alt="Screen Shot 2021-12-29 at 1 40 46 PM" src="https://user-images.githubusercontent.com/54108223/147693499-3a965647-a260-4ca0-b2ea-b0efa9ac6785.png">

It definitely looks better with wider fonts, but these are the only 2 fonts I use regularly, so I think it's best to stick with the default.